eva-accession-release/src/test/java/uk/ac/ebi/eva/accession/release/io/MergedVariantMongoReaderTest.java

```diff
@@ -162,32 +160,24 @@ public void checkMergedInto() throws Exception {
                            .getSourceEntries()
                            .stream()
                            .allMatch(e -> ID_1_MERGED_INTO.equals(e.getAttribute(MERGED_INTO_KEY))));
-
-        assertTrue(variants.get(ID_2)
-                           .getSourceEntries()
-                           .stream()
-                           .allMatch(e -> ID_2_MERGED_INTO.equals(e.getAttribute(MERGED_INTO_KEY))));
     }
 
     @Test
     public void checkAlleles() throws Exception {
         Map<String, Variant> variants = readIntoMap();
-        assertEquals(3, variants.size());
+        assertEquals(EXPECTED_MERGED_VARIANTS, variants.size());
 
         assertEquals("G", variants.get(ID_1_A).getReference());
         assertEquals("A", variants.get(ID_1_A).getAlternate());
 
         assertEquals("G", variants.get(ID_1_T).getReference());
         assertEquals("T", variants.get(ID_1_T).getAlternate());
-
-        assertEquals("T", variants.get(ID_2).getReference());
-        assertEquals("G", variants.get(ID_2).getAlternate());
     }
 
     @Test
     public void includeValidatedFlag() throws Exception {
         assertFlagEqualsInAllVariants(CLUSTERED_VARIANT_VALIDATED_KEY, false);
-        assertFlagEqualsInAllVariants(SUBMITTED_VARIANT_VALIDATED_KEY, false);
+        assertFlagEqualsInAllVariants(SUBMITTED_VARIANT_VALIDATED_KEY, true);
```

Comment:
Just to avoid confusion, could this incidental code change (to support the data) not relevant to the PR changes be avoided by modifying the test data?